Java/Heapsort.java

```diff
@@ -0,0 +1,73 @@
+
+class HeapSort {
+    public int[] A;
+    public int heapSize; // mostly will give the index of last element in heap
+    // constructor
+
+    void Heap(int[] array)
+    {
```

Comment:
Use the same style throughout.
```
func () {

}
```

And use the this case - maxHeapify throughout for the methods. The Class name can be MergeSort but the methods should follow the naming convention of methods.